sqalx.go

```diff
@@ -30,6 +30,10 @@ type Node interface {
 	Close() error
 	// Begin a new transaction.
 	Beginx() (Node, error)
+	// Begin a new transaction using the provided context and options.
+	// Note that the provided parameters are only used when opening the first transaction,
```

Comment:
I'm wondering if this could be confusing to users of the library?
That said, I don't see any alternative to ignoring the parameters. Returning an error would make the addition of the func quite useless.